release/stable/kali-rolling/docker/Dockerfile

```diff
@@ -0,0 +1,80 @@
+# Docker image file that describes an Kali Rolling image with PowerShell installed from tar.gz
```

Comment:
Can you add the image to the CI yml so it will be tested?

Reply:
This is blocking.  The file is here:  https://github.com/PowerShell/PowerShell-Docker/blob/master/vsts-ci.yml

Reply:
- Not Applicable for now since its not officially supported by Microsoft yet.